### ISSUE TITLE
Projects do not build

### ISSUE THREAD

**Issue by felixjones** [kierenj/0x10c-DevKit#184]
On a standard Windows 7 64bit setup with a clean devkit install, creating a C project with the code:

int *screen = 0x8000;

int main()
{
    print("Hello");
}

void print(char _msg)
{
    while (_msg != 0)
    {
        *screen = *msg + 0xf000;
        msg++;
        screen++;
    }
}

And adding bootstrap.dasm16, building the source does nothing.
The output window states "Building solution..." and nothing happens. Clicking build multiple times, or clicking run multiple times, just produces additional lines of "Building solution..."

64 bit Windows 7 Professional
8 GB RAM
Intel i5 Quad 3.3GHz
Anti virus turned off

**Comment by kierenj** [comment]
Hi - sorry you're having a problem!  Currently looking into something similar (though not identical) under another issue here.  Could you try something for me please?

Could you extract this in the "plugins/dcputoolchain" subfolder of the installation - https://dl.dropbox.com/u/18385630/Devkit.DCPUToolchainPlugin.zip - then right-click on the .dll, Properties, and click Unblock button if present.  Then, start Devkit and try the build again - are there any additional messages output?

Thanks!

**Comment by felixjones** [comment]
I extracted it straight in, I did need to unblock, started again and the problem continues on the same solution and a fresh solution, no additional messages.

I am a programmer myself if some debugging on my end would help find the solution.

**Comment by kierenj** [comment]
Would it be possible to zip up your solution, and attach it here or send it, so I can try here?  I did copy and paste your code, but no luck.

You could also try copying the code from bootstrap.dasm16 into a file in the same directory as your .C file, and add that one instead (is the project on a different drive than bootstrap.dasm16?)

**Comment by felixjones** [comment]
Here you are:
https://dl.dropbox.com/u/4803272/devkit-184-1.zip

I'll try copying the bootstrap code, however I did physically move the boostrap into the same location.

Within my project the bootstrap and c source file are in the same location, you'll see in the zip I uploaded.

Another interesting effect is that the Symbols/Navigator list shows up blank.
I'll try this on a second machine and report any similar errors if they come up.

**Comment by kierenj** [comment]
Great!  I got more output from my version, oddly:

Error   Problem parsing DTCC error.  Error line from DTCC: error at line 21: syntax error, unexpected $end, expecting NEWLINE or TEXT   E:\dev\0x10c\Projects\issue184\CHelloWorld\entry.c  0

Add a newline to the end of your C file, and try again ;)  Would you like to raise it as a bug at DCPUToolchain project, or would you like me to?

**Comment by felixjones** [comment]
I can't believe I didn't try that.
You should file the bug report as it's your product using the software.

Thanks for the solution, I'll have to try and keep this in mind. If the DCPUToolchain team don't repair it by next release may I suggest auto line insert? Some instances of Microsoft Visual Studio do this, I wonder if it's a compiler-wide issue.

Thank you again.

**Comment by kierenj** [comment]
Done, thanks for bearing with me on that one.  https://github.com/DCPUTeam/DCPUToolchain/issues/138
